nifty/pipeline/objectoriented/GetConfig.py

```diff
@@ -60,6 +60,22 @@ def __init__(self, args, script):
 
         self.makeConfig()
 
+    def checkConfigExists(self, configFile):
+        """
+        Checks that a config file exists and if not, sets Nifty to use default configuration.
+        """
+        if not os.path.exists(configFile):
+            shutil.copy(self.RECIPES_PATH+'defaultConfig.cfg', configFile)
```

Comment:
It's good practice to use `os.path.join()` to create file paths. Applicable to other places.

Reply:
Very good catch, thank you. Maybe not doing this inadvertently breaks support for Windows users... I've fixed the calls in GetConfig.py in ad9c64d and [created an issue](https://github.com/Nat1405/Nifty4Gemini/issues/8) to go through the rest of the code base to fix those calls.